### PR TITLE
fix(build): Fix no magic numbers lint

### PR DESCRIPTION
**Pull request by mlipardo**

### CODE REVIEW

File: libs/reactstrap/src/utils.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import cssModule from '@cwds/core/dist/bootstrap-cares.module.css'
 import { Util } from 'reactstrap'
 
-Util.keyCodes.p = 80
-Util.keyCodes.n = 78
-Util.keyCodes.home = 36
-Util.keyCodes.end = 35
+Util.rotation = {
```
Comment (eckdanny-osi) [inline libs/reactstrap/src/utils.js]: if we're going to have an enum of literals, can we use `ALL_CAPS` like:

```js
ROTATION = {
  QUARTER: 90,
  HALF: 180,
  THREE_QUARTER: 270,
  FULL: 360
}
```

and

> plz consistent `quarter` vs `fourths`

and probably don't need a `full`.


maybe use `RIGHT`, `LEFT`, `FLIP` ?

File: libs/datagrid/src/DataGridConfig.jsx [hunk 2]
```diff
@@ -6,6 +6,7 @@ import LoadingText from './LoadingText'
 import NoDataText from './NoDataText'
 import PagerButtonFactory from './PagerButtonFactory'
 
+/* eslint-disable no-magic-numbers */
```
Comment (eckdanny-osi) [inline libs/datagrid/src/DataGridConfig.jsx]: 👎

File: libs/datagrid/src/DataGridConfig.jsx [hunk 3]
```diff
@@ -6,6 +6,7 @@ import LoadingText from './LoadingText'
...
 const DataGridDefaults = {
   NextComponent: PagerButtonFactory({ rotation: 270 }),
```
Comment (eckdanny-osi) [inline libs/datagrid/src/DataGridConfig.jsx]: b/c here

File: libs/datagrid/src/Pagination.jsx [hunk 4]
```diff
@@ -1,13 +1,13 @@
 import React from 'react'
 import classnames from 'classnames'
 import PaginationRT from 'react-table/lib/pagination'
-import { Input } from '@cwds/reactstrap'
+import { Input, Util } from '@cwds/reactstrap'
```
Comment (eckdanny-osi) [inline libs/datagrid/src/Pagination.jsx]: 😐 twould be better for the `Util` namespace for `ROTATION` and `KEY_CODES` to exist outside of `@cwds/reactstrap`, but that's out of scope for this story. OK to take on a little debt there, I'll track it in JIRA.

File: .eslintrc.js [hunk 5]
```diff
@@ -19,31 +19,43 @@ module.exports = {
...
+    'no-implicit-coercion': [
+      2,
+      {
+        allow: ['!!'],
+      },
+    ],
+    'no-magic-numbers': [
+      2,
+      {
+        ignore: [-1, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12],
```
Comment (NagaPradeepGajula-osi) [inline .eslintrc.js]: why are we ignoring all these?

File: .eslintrc.js [hunk 6]
```diff
@@ -19,31 +19,43 @@ module.exports = {
...
+      },
+    ],
     'no-useless-concat': [2],
     'no-var': [2],
     'prefer-const': [2],
     'prefer-template': [2],
-    'radix': [2, 'as-needed'],
+    radix: [2, 'as-needed'],
```
Comment (NagaPradeepGajula-osi) [inline .eslintrc.js]: missing quotes accidental?

Comment (eckdanny-osi) [reply .eslintrc.js]: guessing its prettier auto-formatting. This diff is pretty noisy with it 😒

File: .eslintrc.js (hunk 5 above)
Comment (eckdanny-osi) [reply .eslintrc.js]: `[1..12]` for grid system props. These are declarative and value entities (not magic numbers, e.g.; keycodes, http_status, ansi codes...) Obfuscating declarative value entities behind variables _decreases_ readability and degrades the signal-to-noise ratio we want from our lint routine

File: libs/reactstrap/src/utils.js (hunk 1 above)
Comment (eckdanny-osi) [reply libs/reactstrap/src/utils.js]: 🤕 I'm not so hot on adding a new member to `ReactStrap::Util`

File: .eslintrc.js (hunk 5 above)
Comment (karthik-sista) [reply .eslintrc.js]: The team agreed on this.

**Pull request merged**